Mage.Tests/src/test/java/org/mage/test/cards/enchantments/CaseTest.java

```diff
@@ -0,0 +1,103 @@
+package org.mage.test.cards.enchantments;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+public class CaseTest extends CardTestPlayerBase {
+
+    @Test
+    public void test_CaseOfTheBurningMasks() {
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain", 6);
+        addCard(Zone.BATTLEFIELD, playerA, "Aminatou, the Fateshifter");
+        addCard(Zone.HAND, playerA, "Case of the Burning Masks");
+        addCard(Zone.HAND, playerA, "Lightning Bolt", 3);
+        addCard(Zone.HAND, playerA, "Impact Tremors");
+        addCard(Zone.HAND, playerA, "Goblin Chainwhirler");
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Lightning Bolt", playerB);
+        waitStackResolved(1, PhaseStep.PRECOMBAT_MAIN);
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Case of the Burning Masks");
+        waitStackResolved(1, PhaseStep.PRECOMBAT_MAIN, playerA);
+
+        checkPermanentCount("llanowar elves destroyed", 1, PhaseStep.POSTCOMBAT_MAIN, playerB, "Llanowar Elves", 0);
```

Comment:
what's going on here? I don't see this permanent was added nor targeted

Reply:
Ah, I was trying to use a target for the ETB ability for the Case. It looks like the test player doesn't currently handle `TargetOpponentsCreaturePermanent`. I'll remove.

Reply:
Well that's weird- I would like to find out what's the deal with that, but not needed for this PR so that can be a separate investigation later.

Reply:
Cause wait stack for all (not a) and call player b check after it

Reply:
I don't recommends to run commands and checks for diff players at the same priority/phase (cause it can be executed in “wrong” timing for a dev). Only few check commands support checking for another player (see command param for it).

If you need debug or something special to execute then use runCode.

Reply:
If test player doesn't support target type then it will fail with unsupport error. If you can’t setup target choice then something wrong with commands order (see above about diff players and priority).